src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,28 @@
+package duke;
+
+import java.time.format.DateTimeParseException;
+import java.util.Scanner;
+
+public class Duke {
```

Comment:
Very nice adherance to the bracket coding style requirements in the main function. Good job!